src/components/reference/Reference.js

```diff
@@ -85,87 +85,87 @@ export default class Reference extends React.Component {
     let isDisabled = this.props.layoutType === layoutTypes.REFERENCE;
     return (
       <div>
-      {!isDisabled ? 
-      <React.Fragment>
-        <Tooltip title="Reference" placement="bottom-start">
-          <IconButton
-            id="ref-btn"
-            className="header-btn d-none d-md-block"
-            aria-haspopup="true"
-            onClick={this.props.handleReferenceToggle}
-            style={style}>
-            <Icon style={{ color: '#fff' }} className="material-icons">help</Icon>
-          </IconButton>
-        </Tooltip>
-        <Drawer
-          anchor="right"
-          id="reference-drawer"
-          variant="persistent"
-          className={!this.props.referenceOpen ? 'd-none' : ""}
-          open={this.props.referenceOpen}>
-          <Tabs
-            id="reference-tabs"
-            fullWidth={true}
-            value={this.state.value}
-            onChange={this.handleChange} >
-            <Tab
-              icon={<Icon className="material-icons geometry">category</Icon>}
-              label="GEOMETRY"
-              value='a' />
-            <Tab
-              icon={<Icon className="material-icons color-change">bubble_chart</Icon>}
-              label="TRANSFORMATIONS"
-              value='b' />
-            <Tab
-              icon={<Icon className="material-icons animation-ref">zoom_out_map</Icon>} //swap_horiz control_camera category
-              label="ANIMATIONS"
-              value='c' />
-            <Tab
-              icon={<Icon className="material-icons geometry">widgets</Icon>}
-              label="GROUPS"
-              value='d' />
-            <Tab
-              style={{ background: "green", color: "white" }}
-              icon={<Icon className="material-icons">open_in_new</Icon>}
-              label="OPEN IN NEW TAB"
-              value='n'
-              onClick={this.handleOpen} />
-            <Tab
-              style={{ background: "red", color: "white" }}
-              icon={<Icon className="material-icons">close</Icon>}
-              label="CLOSE"
-              value='x'
-              onClick={() => {
-                this.props.handleReferenceToggle();
-                this.setState({ value: 'a' });
-              }}/>
-          </Tabs>
+        {!isDisabled ?
```

Comment:
When doing a really long ternary like this I usually like to put the "?" and ":" on a new line. You can also use "&&" after the cond, rather than using null. It's a style choice but wanted to give some feedback.